test/other/metadce/test_metadce_cxx_ctors1.jssize

```diff
@@ -1 +1 @@
-26205
+26326
```

Comment:
What is the cause of this change?

Reply:
I found a way to avoid all code size regressions.